canonical-websites/www.ubuntu.com
# Update Vanilla and optimise the minified CSS

## Done
Reduced the CSS from 353kb to 215kb (~40%) by updating Vanilla to 1.8.0 and removing SourceMaps from the minified output/ 

## QA
- Check out this feature branch
- **Run `./run clean`**
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Load the site and see that the CSS is around 215kb
- Click through the site and see that the main pages render nicely
- Check small screen navigation works
